apps/api/src/exception-filter.ts

```diff
@@ -0,0 +1,87 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpException, HttpStatus } from '@nestjs/common';
+import { Response } from 'express';
+import { randomBytes } from 'crypto';
+import { PinoLogger } from '@novu/application-generic';
+
+@Catch()
+export class AllExceptionsFilter implements ExceptionFilter {
+  constructor(private readonly pinoLogger: PinoLogger) {}
+  catch(exception: unknown, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+    const request = ctx.getRequest<Request>();
+
+    const { status, message } = this.getResponseMetadata(exception);
+    const responseBOdy = this.buildResponseBody(status, request, message, exception);
+
+    response.status(status).json(responseBOdy);
+  }
+
+  private buildResponseBody(status: number, request: Request, message: string | object | Object, exception: unknown) {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add a response type to all methods‏